src/routes/controllers/room.Controller.js

```diff
@@ -74,6 +74,15 @@ exports.makeRoom = async (req, res, next) => {
   }
 };
 
+exports.getRooms = async (req, res, next) => {
+  try {
+    const rooms = await BattleRoom.find();
+    res.status(200).send({ rooms });
```

Comment:
```suggestion
    const rooms = await BattleRoom.find();
    
    return res.status(200).send({ rooms });
```
개행과 return을 추가 해주는 것은 어떨까요?😄